spec.html

```diff
@@ -24799,7 +24799,7 @@ <h1>The Global Object</h1>
     <li>is created before control enters any execution context.</li>
     <li>does not have a [[Construct]] internal method; it cannot be used as a constructor with the `new` operator.</li>
     <li>does not have a [[Call]] internal method; it cannot be invoked as a function.</li>
-    <li>has a [[Prototype]] internal slot whose value is implementation-dependent.</li>
+    <li>has a [[Prototype]] internal slot whose value is implementation-dependent but the prototype chain includes %Object.prototype%.</li>
```

Comment:
One of the challenges here might be that `%Object.prototype%` is a Realm-specific value, but i believe the WindowProxy causes the *current* realm's `Object.prototype` to report as being in the prototype chain.

Reply:
[`WindowProxy.[[GetPrototypeOf]]()`](https://html.spec.whatwg.org/multipage/window-object.html#windowproxy-getprototypeof) returns `null` for cross‑realm calls.

Reply:
Right - and `null` doesn't have `Object.prototype` in its prototype chain.